src/liquid/components/ld-circular-progress/readme.md

```diff
@@ -273,6 +358,91 @@ You can change the size of the circular progress bar. However, with a smaller si
   <ld-typo id="seo" variant="label-s">SEO</ld-typo>
 </div>
 
+<!-- React component -->
+
+<style>{`
+  .report {
+    display: grid;
+    place-items: center;
+    gap: var(--ld-sp-4);
+  }
+  .report ld-circular-progress {
+    --ld-circular-progress-size: 4rem;
+  }
+  .report-value {
+    transform: scale(0.8);
+  }
+`}</style>
```

Comment:
I did it the same way first, but wasn't really happy with that. Nobody really does it that way with React, I guess. I ended up with assigning a styles object to a variable and assigning that variable to the `style` prop of the components. Maybe you wanna do it the same way, here?

```jsx
const reportStyles = {
  display: 'grid'
  placeItems: 'center'
  gap: 'var(--ld-sp-4)'
}

// and so on ...

return (
  <>
    <div style={reportStyles}>
      {/* ... */}
```